kagi/templates/kagi/base.html

```diff
@@ -14,16 +14,14 @@
 </script>
 {{ block.super }}
 
-{% comment %}
-TODO: Handle browser not supporting navigator.credentials API.
-{% endcomment %}
-<div id="kagi-not-defined-error" style="display: none">
+<div id="webauthn-undefined-error" style="display: none">
     {% blocktrans %}
     You must be using a compatible browser to use a WebAuthn security key.
-    The following browsers are known to be compatible
+    The following browsers are known to be compatible:
     {% endblocktrans %}
     <ul>
-      <li>{% trans 'Chrome 41 and higher' %}
+      <li>{% trans 'Firefox 67 and higher' %}
+      <li>{% trans 'Chrome 67 and higher' %}
```

Comment:
Shall we rather add a link to caniuse.com?

Reply:
Yes, I think you're probably right. Will change that.

Reply:
Changed to a link in https://github.com/justinmayer/kagi/pull/36/commits/79bbaf038d560b8cd280027d1dbd553f6e947f33.